lib/cdo/ci.rb

```diff
@@ -1,34 +1,31 @@
 require 'set'
 
-CIRCLE_NODE_TOTAL = ENV['CIRCLE_NODE_TOTAL'].to_i
-CIRCLE_NODE_INDEX = ENV['CIRCLE_NODE_INDEX'].to_i
-
-module CircleUtils
-  # Checks the HEAD commit for the current circle build for the specified tag,
+module CI
```

Comment:
I did kinda like the `Utils` clarification here; made it clear that this code isn't intended to actually execute Circle/CI concerns, but just some tools to help with minor details of that execution. Of course, `CIUtils` is less than ideal; what do you think about something like,
```suggestion
module CI
  module Utils
```

Reply:
CI::Utils is a nice compromise (you guessed correctly: renamed because I just couldn't stand the visuals on CIUtils).

Reply:
Renamed (likely broke the build during the mass-rename tho, will check later)